### DISCUSSION TITLE
Feature Request: Configurable Logging Stream

### DISCUSSION THREAD

**Discussion by SamHalvoe** [pschatzmann/arduino-audio-driver#39] [Ideas]
Hi
Thanks for this cool library.
For my project I want to log warnings and errors to a file and I'm using your Audio Driver and Audio Tools library.
Therefore it would be great, if the stream for logging could be configured by the user like with your Audio Tools library:
`AudioLogger::instance().begin(file, AudioLogger::Warning);`

**Comment by pschatzmann** [comment]
I committed the following new functionaity: It's possible to configure the logger with

    AudioDriverLogger.begin(Serial,AudioDriverLogLevel::Info );

**Comment by SamHalvoe** [comment]
Thank you very much.
